docs/develop/configure-your-add-in-to-use-a-shared-runtime.md

```diff
@@ -59,6 +62,7 @@ Follow these steps for a new or existing project to configure it to use a shared
          <Runtime resid="Taskpane.Url" lifetime="long" />
```

Comment:
Can you indent Runtimes section? It looks like it's at the same level as Host.

Reply:
Yes, fixed!